File: docs/user/en/query/ordering/examples/constant/attribute-set-exact.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeStartsWith("code", "lenovo")
				),
				orderBy(
					attributeSetExact(
						"code", 
						"lenovo-tab-m8-3rd-generation", 
						"lenovo-yoga-tab-13", 
						"lenovo-tab-m10-fhd-plus-3rd-generation-1"
					)
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/constant/attribute-set-exact.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeStartsWith("code", "lenovo")
        	),
        	OrderBy(
        		AttributeSetExact(
        			"code", 
        			"lenovo-tab-m8-3rd-generation", 
        			"lenovo-yoga-tab-13", 
        			"lenovo-tab-m10-fhd-plus-3rd-generation-1"
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);